frontend/src/runtime-config.ts

```diff
@@ -4,8 +4,9 @@ import type { CreateClientConfig } from './client/client.gen'
 export const createClientConfig: CreateClientConfig =(config) => {
   const isServer = typeof window === 'undefined'
 
-  const baseURL =
-    process.env.NEXT_PUBLIC_BACKEND_BASE_URL ?? 'http://localhost:8000'
+  const baseURL = isServer
```

Comment:
You are probably running into this error because you are not using the `client`.

https://github.com/A3copilotprogram/plg_6/pull/15#issuecomment-3338140315

Reply:
I can remove this in this PR but this was an issue I ran into even before implementing podcast. I couldn't log in or sign up. _(PS: I'm running the project with docker)_